vgen/src/main/resources/io/gapi/vgen/java/java_gapic.yaml

```diff
@@ -1,8 +1,10 @@
 type: io.gapi.vgen.ConfigProto
 language: java
-language_provider: io.gapi.vgen.java.JavaGapicLanguageProvider
-snippet_files:
-- main.snip
-- settings.snip
-fragment_files:
-- fragment.snip
+templates:
+- language_provider: io.gapi.vgen.java.JavaGapicCodeLanguageProvider
+  snippet_files:
+  - main.snip
+  - settings.snip
+- language_provider: io.gapi.vgen.java.JavaGapicFragmentLanguageProvider
+  snippet_files:
+  - fragment.snip
```

Comment:
I'm not sure either, but changing this would be out-of-scope for this PR. The fragment file was previously declared here, and we're just changing how it's done. I'm ramping up on vKit snippet gen for Python, and hope to better understand the use cases in a few days.